build.sbt

```diff
@@ -26,55 +26,64 @@ def docDepSettings = Seq(
   previewSite := {}
 )
 
-lazy val coulomb = (project in file("coulomb"))
+lazy val coulomb = crossProject(JVMPlatform, JSPlatform)
+  .crossType(CrossType.Full)
```

Comment:
Most projects can be fully cross built, but `coulomb` needs platform specific implementations of `UnitTypeName`